controls/combobox/commands.md

```diff
@@ -24,6 +28,32 @@ The example below shows both cases, the default `ClearSelectionCommand` executio
 
 ![ComboBox ClearSelectionCommand](images/combobox-commands.gif)
 
+## Remove Token Command
```

Comment:
The suggestions from the `controls/autocomplete/commands.md` article apply here too.